cockatrice/src/settings/cache_settings.cpp

```diff
@@ -252,8 +252,8 @@ SettingsCache::SettingsCache()
     chatMentionColor = settings->value("chat/mentioncolor", "A6120D").toString();
     chatHighlightColor = settings->value("chat/highlightcolor", "A6120D").toString();
 
-    zoneViewGroupBy = settings->value("zoneview/groupby", 2).toInt();
-    zoneViewSortBy = settings->value("zoneview/sortby", 1).toInt();
+    zoneViewGroupByIndex = settings->value("zoneview/groupby", 1).toInt();
```

Comment:
does this change the default value?

Reply:
`SortByType` is at enum position 2 and `QComboBox` index 1, so the default is still `Sort by Type`